db/deletion.py

```diff
@@ -0,0 +1,25 @@
+"""Force django to use the ON DELETE/ON UPDATE clauses when making foreign keys
+NOTE: delete hooks for cascaded objects WILL NOT be run
+"""
+from django.db.models.deletion import DO_NOTHING
+
+
+class DatabaseOnDelete:
+
+    def __init__(self, clause='NO ACTION'):
+        self.clause = clause
+
+    def __call__(self, collector, field, sub_objs, using):
+        pass
+
+    def __eq__(self, other):
+        return DO_NOTHING == other or (isinstance(other, DatabaseOnDelete) and other.clause == self.clause)
```

Comment:
~~Why is `DO_NOTHING` always equal?~~
Oh I see, [part of that dirty hack](https://github.com/django/django/blob/master/django/db/models/deletion.py#L148). I think that's worth a comment?